tests/jobs/scala/src/main/scala/KafkaCassandraStream.scala

```diff
@@ -0,0 +1,137 @@
+import java.util.UUID.randomUUID
+
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.mllib.random.RandomRDDs.normalRDD
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.streaming.dstream.ConstantInputDStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import scopt.OptionParser
+
+import scala.math.max
+import scala.util.Random
+
+
+case class Employee(
+                     id: String,
+                     first_name: String,
+                     last_name: String,
+                     email_address: String,
+                     address: String,
+                     city: String
+                   )
+
+/**
+  * Application that generates random records and streams them to Cassandra.
+  *
+  * Usage:
+  * {SPARK} {ARGS} {PATH_TO_ASSEMBLY}
+  * where {SPARK} is either `dcos spark run` or `./bin/spark-submit`
+  * {ARGS} are the configuration arguments to Spark.
+  * {PATH_TO_ASSEMBLY} is the location of the snapshot jar containing this application
+  *
+  * Example:
+  * dcos spark run --submit-args='\
```

Comment:
This should include the other spark options -- like limiting the number of executors.